app/components/Product/product-details-description.tsx

```diff
@@ -0,0 +1,73 @@
+import { Button } from "../ui/button";
+import { Separator } from "../ui/separator";
+
+export default function ProductDetailsDescription() {
```

Comment:
Możesz podzielić na dwa dodatkowe komponenty jeden do wyboru rozmiarów a drugi do wyboru liczy zamawianych produktów (ale to dopiero jak będzie logika)